### PR TITLE
Implemented the clean up of the download folder on startup

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@donnapep @stulees please review. cheers

### CODE REVIEW

File: app/helpers/file-system.js [hunk 1]
```diff
@@ -109,6 +109,14 @@ module.exports = {
     });
   },
 
+  cleanupDownloadFolder: function() {
+    fs.emptyDir(config.downloadPath, (err) =>{
+      if (err && config.debugging) {
+        console.error(err);
```
Comment (rodrigopavezi) [inline app/helpers/file-system.js]: @donnapep is that okay if I leave this change here for you then? cheers

Comment (donnapep) [inline app/helpers/file-system.js]: @rodrigopavezi Sure, OK.

Comment (rodrigopavezi) [inline app/helpers/file-system.js]: @stulees FYI. cheers

**Comment by stulees** [comment]
@rodrigopavezi 👍

**Pull request merged**